.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+   "presets": ["@babel/preset-env", "@babel/react"],
+   "plugins": [
+      "@babel/plugin-proposal-class-properties",
+      "@babel/plugin-proposal-export-default-from"
```

Comment:
As discussed, I would not use this plugin, as it is still a Stage 0 proposal within the tc39 process (as opposed to class properties, which is a stage 3 proposal)